src/Compatibility/Core/src/iOS/Forms.cs

```diff
@@ -637,8 +639,12 @@ public async Task<Stream> GetStreamAsync(Uri uri, CancellationToken cancellation
 				}
 			}
 
-#if __MOBILE__
+#if MACCATALYST
```

Comment:
Every other check in this PR uses `#if MACCATALYST || __MACCATALYST__` but this line only uses `#if MACCATALYST`. Is that an issue?